### DISCUSSION TITLE
Direct and extended predicted TF-to-region-to-gene links.

### DISCUSSION THREAD

**Discussion by w1973145618** [aertslab/scenicplus#396] [Q&A]
Hello, I have a bit of confusion about the Direct and Extended Predicted TF-to-region-to-gene links. I do not understand why there is a distinction between Direct and Extended, what is the difference between them?
Furthermore, the peak data detected by ATAC-seq can be the transcription start site or enhancer region. I don't understand which step excludes peaks at the transcription start site.

I need  help.

**Comment by SeppeDeWinter** [comment]
Hi @w1973145618 

For you question related to "direct" and "extended" annotations, please see: https://github.com/aertslab/scenicplus/discussions/373.

For your second question, those peaks are in general not excluded from the analysis.

Best,

Seppe
